style.css

```diff
@@ -1823,60 +1906,321 @@ a {
     color: #ff6b00;
     border: none;
+    flex: 1;
+    padding: 12px;
   }
 
   .wrapper-B {
     margin: 0 auto;
-    width: 1440px;
-    height: 44px;
+    width: 375px;
+    height: 40px;
     background-color: white;
     display: flex;
+    flex-direction: column;
     justify-items: right;
     border-top-style: solid;
     border-top-color: #dfe1e6;
     border-top-width: 1px;
+    flex: 1;
   }
 
   .social-group-end {
     width: 307px;
     height: 24px;
-    background-color: white;
     color: white;
-    justify-items: center;
-    gap: 30px;
     display: flex;
+    justify-content: space-around;
     align-items: center;
-    margin-left: 600px;
-    padding-top: 12px;
+    flex: 1;
+    margin-left: 45px;
+  }
+
+  .rectangle {
+    height: 3px;
+    width: 100px;
+    background-color: #555;
+    margin-left: 145px;
+  }
+
+  .mobile-menu {
+    display: none;
+    margin: 0 auto;
+    flex-direction: column;
+    justify-content: flex-end;
+    width: 375px;
+    height: 812px;
+    float: inline-start;
+    background: #3c3a39;
+    box-shadow: 1px 48px 48px rgba(37, 47, 137, 0.08);
+  }
+
+  .cross {
+    display: flex;
+    justify-content: flex-end;
+    padding: 37px;
+  }
+
+  .entries {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    display: flex;
+    align-items: center;
+    color: #fff5e1;
+    padding: 15px;
+    border-bottom-style: solid;
+    border-bottom-width: 1px;
+    border-color: #6f6c6b;
+    margin: 12px 24px 24px 24px;
+  }
+
+  .contact-entries {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    display: flex;
+    align-items: center;
+    color: #fff5e1;
+    padding: 15px;
+    border-bottom-style: solid;
+    border-bottom-width: 1px;
+    border-color: #6f6c6b;
+    margin: 12px 24px 324px 24px;
+  }
+
+  .rectangle-white {
+    margin: 0 auto;
+    background-color: #e5e5e5;
+    width: 375px;
+    height: 57px;
+    display: flex;
+    align-items: flex-end;
+    justify-content: center;
+  }
+
+  .line-decoration {
+    margin: 0 auto;
+    width: 100px;
+    height: 1px;
+    border-bottom-style: solid;
+    border-bottom-width: 3px;
+    border-bottom-color: #6f6c6b;
+    margin-bottom: 5px;
+  }
+
+  .overlay {
+    position: fixed;
+    width: 375px;
+    height: 600px;
+    top: 0%;
+    left: 10%;
+    right: 10%;
+    margin-right: auto;
+    margin-left: auto;
+    background: #979493;
+    display: none;
+    z-index: 1;
+    mix-blend-mode: multiply;
+  }
+
+  .container-pop-up {
+    box-sizing: border-box;
+    padding: 16px;
+    gap: 24px;
+    position: fixed;
+    width: 343px;
+    height: 718px;
+    left: calc(50% - 343px / 2);
+    top: 10px;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    z-index: 2;
+  }
+
+  .wrapper-0 {
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
As [the last code reviewer suggested](https://github.com/edahigure/Portfolio-setup-and-mobile-first/pull/13#discussion_r1002221758) kindly fix the position and the size of your popup windows on both versions mobile and desktop. :pray: :pray: 
- Please apply the changes in your desktop version and check it with your browser window zoom at 100%. :eyes: :eyes: 
Your design
![image](https://user-images.githubusercontent.com/98363075/197315046-20574107-376d-4d6a-ac0d-576b05be13ab.png)
Figma design
![image](https://user-images.githubusercontent.com/98363075/197315091-ea87a0dc-56b6-47b1-87f3-4676bf2a5e67.png)

- The design in your mobile version is totally different, kindly place the elements in the same way that the Figma template. :+1: :smiley: :smiley: 
Your design.
![image](https://user-images.githubusercontent.com/98363075/197315141-d1e6c740-f9c4-496b-b23d-332b83c98ae6.png)

Figma design
![image](https://user-images.githubusercontent.com/98363075/197315187-3d0cd681-8802-4ee9-a829-ea8e2851c04d.png)